### ISSUE TITLE
Mini player doesn't work (again)

### ISSUE THREAD

**Issue by spencermathews** [radiant-player/radiant-player-mac#683]
I'm aware that this issue has been raised in the past, around 5 years ago. Those issues were apparently resolved but somehow I'm experiencing the exact same issue. Namely, the mini player is stuck at "No song currently playing" despite the fact that it actually is. The standard interface seems to work fine.

Radiant Player 1.12.0
macOS Mojave 10.14.6

**Comment by tjanson** [comment]
Just FYI: The default answer for a while has been to check out https://github.com/MarshallOfSound/Google-Play-Music-Desktop-Player-UNOFFICIAL- instead — it's pretty much functionally identical, and under much more active development.

**Comment by jacobwgillespie** [comment]
Yep, see https://github.com/radiant-player/radiant-player-mac/issues/675#issuecomment-472638065 for context
